src/base/utils/random.cpp

```diff
@@ -30,146 +30,16 @@
 
 #include <random>
 
-#include <QtLogging>
 #include <QtSystemDetection>
 
-#if defined(Q_OS_WIN)
-#include <windows.h>
-#include "base/global.h"
-#include "base/utils/os.h"
-#elif defined(Q_OS_LINUX)
-#include <cerrno>
-#include <cstring>
-#include <sys/random.h>
+#if defined(Q_OS_LINUX)
+#include "randomlayer_linux.cpp"
```

Comment:
I would still use `.h` extension for these files.

Reply:
But they have different private fields and the header will become 'dirty' if all platform specific code are in there.

Reply:
I mean "randomlayer_linux.h" etc.
But it looks strange because of unnamed namespace there.

Reply:
>I mean "randomlayer_linux.h" etc.

Then the header will contain implementations which we previously keep it in .cpp files.
It seems to me, doing it other way will not be as clean as the current.